test/aria/widgets/form/timefield/checkValue/TimeField.js

```diff
@@ -19,16 +19,115 @@ Aria.classDefinition({
     $constructor : function () {
         this.$TemplateTestCase.constructor.call(this);
         this.data = {
-            time1 : null,
-            time2 : null
-
+            time1 : null
         };
+        this.currentValue;
+        this.testValues = [];
+        this.checkValues = [];
+        this.separators = ["/", "\\", "h", ";", ",", ".", "-", ":"];
+        this.am = ["am", "a.m.", "a m"];
+        this.pm = ["pm", "p.m.", "p m"];
+        this.start();
         this.setTestEnv({
             data : this.data
         });
     },
     $prototype : {
+        start : function () {
+            this.check1Digit();
+            this.check2Digits();
+            this.check3Digits();
+            this.check4Digits();
+            this.checkAlphabetic();
+            this.checkValuesWithAMPM();
+        },
+        check1Digit : function () {
+            var hour, result, i;
+
+            // 1 digit = [h]
+            for (i = 0; i < 10; i++) {
+                hour = i + '';
```

Comment:
I would reverse the order of operands here: it would make it more obvious we are casting to a string (and it seems to be more widespread).